Commivoyajer_Framework/Methods/BruteForceMethod.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Commivoyajer_Core.Models;
+
+namespace Commivoyajer_Core.Methods
+{
+    public class BruteForceMethod
+    {
+        public Output GetBruteForceMethod(double[,] dis_matrix) // Сама функция перебора
+        {
+            int[] city_array = new int[dis_matrix.GetLength(0)];
+            int[] answer_array = new int[dis_matrix.GetLength(0)];
+            double minimal_dist = 0;
+            double new_dist = 0;
+
+            for (int i = 0; i < dis_matrix.GetLength(0); i++)
+                city_array[i] = i + 1;
+
+            answer_array = city_array;
+            minimal_dist = CalculateDistance(dis_matrix, city_array);
+
+            while (permNxt(city_array, 0))
+            {
+                new_dist = CalculateDistance(dis_matrix, city_array);
+                if (minimal_dist > new_dist)
+                {
+                    minimal_dist = new_dist;
+                    answer_array = city_array;
+                }
+            }
+
+            // Вывод пути и длины пути в текстбокс
+            // answer_array - is answer
+            return new Output
+            {
+                Sequence = answer_array,
+                JourneyLength = 0,
+                CalculationTime = 0,
+                variantsCount = 0
+            };
+        }
+
+        private int[] Swap(int[] ar, int i, int j) // доп.функция генерации перестановок маршрута
+        {
+            int a = ar[i];
+            ar[i] = ar[j];
+            ar[j] = a;
+            return ar;
+        }
+
+        private bool permNxt(int[] ar, int a) // доп.функция генерации маршрута
```

Comment:
The code should be self-described. It will be better to use full variable names and functions